src/Lykke.Quintessence.Domain/Services/AddAddressResult.cs

```diff
@@ -0,0 +1,28 @@
+namespace Lykke.Quintessence.Domain.Services
+{
+    public abstract class AddAddressResult
```

Comment:
I don't understand how it should be used/checked. The callers have to check the type of result, haven't they? Why not use enums/constants?

Reply:
Just for uniformity

Reply:
Redundant complication, in my opinion.